seed/seed16.go

```diff
@@ -61,6 +62,7 @@ type seed16 struct {
 }
 
 func (s *seed16) LoadAssertions(db asserts.RODatabase, commitTo func(*asserts.Batch) error) error {
+	logger.Debugf("loading assertions for UC16/18 system")
```

Comment:
this is clear enough from the logging loadAssertions, no?

Reply:
In general I am trying to add some more debug traces here and there as I think we are not very verbose and sometimes it gets difficult to get good logs from customers. In the end you will not see this in a normal run. But yeah, maybe these two ones are not that useful.